### PR TITLE
[DCOS 46205] Release notes for HDFS 2.5.0-2.6.0-cdh5.11.0

### PR DESCRIPTION
**Pull request by soumasish**
## Description

https://jira.mesosphere.com/browse/DCOS-46205

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+excerpt: Release notes for v. 2.5.0-2.6.0-cdh5.16.3
+title: Release Notes
+menuWeight: 10
+model: /services/hdfs/data.yml
+render: mustache
+---
+## Version 2.5.0-2.6.0-cdh5.16.3
+
+### Bug Fixes
+- This release fixes the bug where quorum would on HDFS when a Journalnode was replaced.
```
[the remark this answers is not in the document]
Comment (soumasish) [reply pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md]: Done

File: pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+- This release fixes the bug where, if HDFS had too many files the block reports to the namenodes failed because the reports were too big. The fix sets the env var `ipc.maximum.data.length` to a higher value (default is 64MB).
```
[the remark this answers is not in the document]
Comment (soumasish) [reply pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md]: Done.

File: pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+render: mustache
+---
+## Version 2.5.0-2.6.0-cdh5.16.3
+
+### Bug Fixes
+- Fix for JournalNodes failing to establish quorum when paxos folder is missing.
+- Hadoop `ipc.maximum.data.length` property is now available in service configuration.
+
+### Improvements
+- A validator has been added to check the package has a unique name and follows package naming guidelines on universe.
```
Comment (samvantran) [inline pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md]: Suggestion: `- Added validation to verify package names are unique and follow Universe guidelines`

File: pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+### Updates
+- Hadoop distribution has been upgraded from cdh5.11 to cdh5.16.1
```
[the remark this answers is not in the document]
Comment (soumasish) [reply pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md]: Thanks, fixed.

File: pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+title: Release Notes
+menuWeight: 10
+model: /services/hdfs/data.yml
+render: mustache
+---
+## Version 2.5.0-2.6.0-cdh5.16.3
+
+### Bug Fixes
+- Fix for JournalNodes failing to establish quorum when paxos folder is missing.
+- Hadoop `ipc.maximum.data.length` property is now available in service configuration.
```
[the remark this answers is not in the document]
Comment (soumasish) [reply pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md]: Thanks, fixed.

File: pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+### Improvements
+- A validator has been added to check the package has a unique name and follows package naming guidelines on universe.
+
+### Updates
+- Hadoop distribution has been upgraded from cdh5.11 to cdh5.16.1
+- libemesos bundle has been upgraded to 1.12
```
[the remark this answers is not in the document]
Comment (soumasish) [reply pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md]: Thanks, fixed.

File: pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md (hunk 3 above)
Comment (soumasish) [reply pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md]: Thanks, fixed.

**Comment by SarahStegall** [comment]
The description of this PR/JIRA ticket is "release notes", yet it appears that this is a new version of the docs. Have changes been made to other pages as well?

File: pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+navigationTitle: Release Notes
+excerpt: Release notes for v. 2.5.0-2.6.0-cdh5.16.3
+title: Release Notes
+menuWeight: 10
+model: /services/hdfs/data.yml
+render: mustache
+---
+## Version 2.5.0-2.6.0-cdh5.16.3
+
+### Feature
```
Comment (samvantran) [inline pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md]: Previous versions phrase this as `### New Features`

File: pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+- Hadoop `ipc.maximum.data.length` property is now available in service configuration.
+
+### Bug Fixes
+- Fix for JournalNodes failing to establish quorum when paxos folder is missing.
+
+### Improvements
+- Added validation to verify package names are unique and follow Universe guidelines.
+
+### Updates
+- Upgraded Hadoop distribution from cdh5.11.0 to cdh5.16.3
```
Comment (samvantran) [inline pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md]: No need to state what the previous version was, it's literally two lines down as a heading

**Pull request closed without merging**